bin/shutter

```diff
@@ -826,7 +837,9 @@ sub STARTUP {
 	$st->{_full}->signal_connect('clicked' => \&evt_take_screenshot, 'full');
 
 	#init menus
-	$st->{_full}->set_menu(fct_ret_workspace_menu(TRUE));
+	if ($x11_supported){
+		$st->{_full}->set_menu(fct_ret_workspace_menu(TRUE));
```

Comment:
Does wayland have no concept of workspaces? Either way, I thought Wnck only lists X11 windows, not workspaces...

Reply:
I'm not sure why, but this line gave an error if run on a Wayland setup. Actually, for some reason when trying to stop Shutter via Ctrl+C from terminal. It prevented Shutter from being stopped.